tests/test_states.py

```diff
@@ -0,0 +1,65 @@
+
+# pylint: disable=protected-access, unused-argument
+
+
+# ------------------------------------------------------------------------------
+#
+def test_pilot_state_value():
+    from radical.pilot.states import _pilot_state_value
+    assert(_pilot_state_value('NEW')                    == 0)
+    assert(_pilot_state_value('PMGR_LAUNCHING_PENDING') == 1)
+    assert(_pilot_state_value('PMGR_LAUNCHING')         == 2)
+    assert(_pilot_state_value('PMGR_ACTIVE_PENDING')    == 3)
+    assert(_pilot_state_value('PMGR_ACTIVE')            == 4)
+    assert(_pilot_state_value('DONE')                   == 5)
+    assert(_pilot_state_value('FAILED')                 == 5)
+    assert(_pilot_state_value('CANCELED')               == 5)
```

Comment:
To be honest, I don't find these tests particularly useful.  The numbers we assign are not really special, so these tests basically assert that `a=b` results in `a=b` - duh ;-)  Not that I have an educated opinion what types of tests would make sense in this context, just saying...

Reply:
Yeah, but this is the first file in issue #1881. And apparently it is the easiest. I agree with you it is a bit weird to test files that create constants only.

Reply:
There are a couple of methods there which are supposed to ensure correct state progression.  Those are rather central to how we define and ensure states.  The `collapse` is tested, we may want to add test to the `progress` versions, too?

Reply:
Okay!

Reply:
Should this still be addressed or  will that go into a different PR?  If the latter, please open a ticket to keep track of it.